### PR TITLE
main/py-pep8: enable py3

### PR DESCRIPTION
**Pull request by fabaff**

### CODE REVIEW

**Comment by jirutka** [comment]
pep8 has been renamed to pycodestyle. See https://github.com/PyCQA/pycodestyle.

**Comment by fabaff** [comment]
Now there is a permission error.

```bash
  File "/usr/lib/python3.5/site-packages/pkg_resources/__init__.py", line 1580, in _get
    with open(path, 'rb') as stream:
PermissionError: [Errno 13] Permission denied: '/usr/lib/python3.5/site-packages/pycodestyle-2.1.0-py3.5.egg-info/PKG-INFO'
```

**Comment by jirutka** [comment]
testing/pep8 is a duplicate of main/pyp-pep8, isn’t it?

**Comment by fabaff** [comment]
Seems so, but not sure. I guess that with `replaces=` we can play it safe.

File: main/py-pep8/APKBUILD [hunk 1]
```diff
@@ -1,3 +1,6 @@
+#
+# Replaced by pycodestyle
```
Comment (fabled) [inline main/py-pep8/APKBUILD]: If it's truly replaced, we should delete the whole APKBUILD.

File: main/pycodestyle/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Contributor: Peter Bui <pnutzh4x0r@gmail.com>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=pycodestyle
+pkgver=2.1.0
+pkgrel=0
+pkgdesc="A Python style guide checker"
+url="http://pep8.readthedocs.io/"
+arch="noarch"
+license="MIT"
+replaces=py-pep8
```
Comment (fabled) [inline main/pycodestyle/APKBUILD]: Should we also add `provides="py-pep8=$pkgver"` ?

File: testing/pep8/APKBUILD [hunk 3]
```diff
@@ -1,3 +1,6 @@
+#
+# Replaced by py-pep8
```
Comment (fabled) [inline testing/pep8/APKBUILD]: Again, let's just remove this instead.

File: main/pycodestyle/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,49 @@
+# Contributor: Peter Bui <pnutzh4x0r@gmail.com>
+# Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
+pkgname=pycodestyle
```
Comment (fabled) [inline main/pycodestyle/APKBUILD]: Could this go in community, or is there good reasons for it to be in main?

**Pull request closed without merging**